Makefile

```diff
@@ -97,38 +97,39 @@ analyse_canary: ## run php-cs-fixer and phpstan (dark-canary)
 ##@ DEPLOY
 
 ACCOUNT=$(shell aws sts get-caller-identity --profile=poser | jq -r '.Account')
```

Comment:
AWS_ACCOUNT_ID ? :P

Reply:
This should be the second line, thus to be able to use `$AWS_PROFILE` instead of hardcoded `poser`

Reply:
fixed